test/FlapperUniV2.t.sol

```diff
@@ -62,6 +62,10 @@ interface GemLike {
     function transfer(address, uint256) external;
 }
 
+interface Univ2FactoryLike {
```

Comment:
nit: we usually capitalize it as `UniV2XXX` in the rest of this repo.

Reply:
Done